tests/test_samples_checker.py

```diff
@@ -15,6 +16,11 @@ def test_get_samples_from_vcf(self):
     def test_get_sample_names(self):
         assert get_sample_names([{'Sample Name': 'S1'}, {'Sample ID': 'S2'}, {'Analysis': ''}]) == ['S1', 'S2']
 
+
+class TestSampleChecker(TestCase):
```

Comment:
Shouldn't all these tests be under `TestSampleChecker` actually?  I'm not sure where the `TestFtpDepositBox` name came from...

Reply:
Probably copy/paste error that I never questioned after ...
Changed it now.